### DISCUSSION TITLE
Link between the database and Framework Symphony

### DISCUSSION THREAD

**Discussion by Myflowerlife-com** [symfony/symfony#49386] [Q&A]
Good morning,
I hope you’re okay?
We have a big problem connecting our database (on IONOS). We use a Symphony framework. We put the correct name of the database, but we always have one mistake:
SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for mysql failed: Name or service not known

How to find the error in its code if there is an error? Thank you
Do you have a lead to help us?

**Comment by brentlobbezoo** [comment]
There's probably something wrong with the host you have specified in your configuration file. Without knowing your configuration, we can't really help. Could you show us, without username and password obviously, your DATABASE_URL in your env file? We can check whether the syntax is correct.

It should look something like this:
`DATABASE_URL="postgresql://app:!ChangeMe!@127.0.0.1:5432/app?serverVersion=14&charset=utf8"`

We can't really help you locate the server or provide you with the correct IP, username, pass, etc. So you have to make sure those values are correct. Just make sure that, if the db is on the same machine as the application, that the host should be localhost (or 127.0.0.1).

**Comment by ghost** [comment]
There's no such thing as Symphony framework.
